packages/babylon-ts-sdk/docs/pegin-flow.md

```diff
@@ -0,0 +1,172 @@
+# Peg-in Flow
+
+This guide walks through creating a peg-in transaction step by step.
```

Comment:
Could we have a more in depth introduction here? What's a peg-in transaction, who creates it, and why?